subprojects/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -532,4 +533,28 @@ public void setArchiveBase(PathBase archiveBase) {
     public Property<Integer> getNetworkTimeout() {
         return networkTimeout;
     }
+
+    /**
+     * Sets if the task will validate the distribution url that has been configured.
+     *
+     * @since 8.2
+     */
+    @Incubating
+    @Option(option = "validate-url", description = "Sets task to validate the configured distribution url")
+    public void setValidateDistributionUrl(Boolean validateDistributionUrl) {
+        this.validateDistributionUrl.set(validateDistributionUrl);
+    }
+
```

Comment:
We don't need to add setters like this.  Everything needs to go through the `getValidateDistributionUrl` below.

```suggestion

```